### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Patfarmurs**
In this Project I :
Use webpack  configuration 
Use ES6 syntax
Add Linters configuration

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
```
Comment (zieeco) [inline Readme.md]: - Please, kindly replace [Your_project_name] with the name of your project as this is a placeholder telling you what to do.

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+To do list structure
+
+> Describe your project in 1 or 2 sentences.
+This project is based on a dayly to do list that provides  a set of features,
+where a user can add and remove something he or she will do on the list.
+
+**[your_project__name]** is a...
```
Comment (zieeco) [inline Readme.md]: - Kindly remove all these placeholders and add some description about your project to give users an overview of the project. The placeholder text are meant for your to add something in place of them

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+#To do list structure
+
+## 🛠 Built With <a name="built-with"></a>
+Html
+CSS and Javascript
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (zieeco) [inline Readme.md]: - Please, remove this placeholder since you've added the tech stack used in your project.

File: Readme.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+    <li>CSS</li>
+    <li>JAVASCRIPT</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (zieeco) [inline Readme.md]: - Let's remove this also as these are instructions for you to follow in adding content to the readme following the guidelines.

File: Readme.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (zieeco) [inline Readme.md:66]: - Let's remove this also as these are instructions for you to follow in adding content to the readme following the guidelines.

File: Readme.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+- Twitter: [@twitterhandle](https://twitter.com/MukulayengeP)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/patrick-m-5601831a1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
Comment (zieeco) [inline Readme.md]: - Let's remove this also as these are instructions for you to follow in adding content to the readme following the guidelines.

File: Readme.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank to Figma for the inspiration.
```
Comment (zieeco) [inline Readme.md]: - Let's remove this also as these are instructions for you to follow in adding content to the readme following the guidelines. Also, you've not used a Figma template for this project, so you should consider updating this to make your redme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Patfarmurs ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Readme.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (Bondok6) [inline Readme.md:182]: - [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

> Also, I suggest removing this `NOTE ...` recommendation from your README file 👍

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Patfarmurs 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**